src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3845,6 +3845,15 @@
   <data name="InvalidOperation_SuppliedInnerMustBeMarkedAggregation" xml:space="preserve">
     <value>Supplying a non-null inner should also be marked as Aggregated.</value>
   </data>
+  <data name="InvalidOperationException_NoGCRegionCallbackAlreadyRegistered" xml:space="preserve">
+    <value>A NoGCRegion callback is already registered.</value>
+  </data>
+  <data name="InvalidOperation_UnableToEnableNoGCRegionCallback_NotStarted" xml:space="preserve">
+    <value>Failed to register a NoGCRegion callback because we are not in a NoGC region.</value>
```

Comment:
Nit: First person is not the style that we use for error messages. There is no error message in this file that uses "I ..." or "We ...".